Source/WebCore/workers/WorkerRunLoop.cpp

```diff
@@ -144,6 +145,7 @@ class RunLoopSetup {
 private:
     WorkerDedicatedRunLoop& m_runLoop;
     IsForDebugging m_isForDebugging { IsForDebugging::No };
+    AutodrainedPool pool;
```

Comment:
This one might be redundant with the one you add below in WorkerDedicatedRunLoop::runInMode.
I would tend to keep only the one in WorkerDedicatedRunLoop::runInMode if possible.